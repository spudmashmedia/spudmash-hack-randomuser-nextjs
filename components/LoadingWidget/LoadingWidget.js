/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Spudmash Media Pty Ltd
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

import React from "react";
import { Paper, CircularProgress } from "@material-ui/core";

let LoadingWidget = props => {
  return (
    <Paper>
      <CircularProgress />
    </Paper>
  );
};

export default LoadingWidget;
