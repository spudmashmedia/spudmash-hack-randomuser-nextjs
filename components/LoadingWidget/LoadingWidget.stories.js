/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Spudmash Media Pty Ltd
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

import React from "react";
import LoadingWidget from "./LoadingWidget";

export default {
  component: LoadingWidget,
  title: "LoadingWidget",
  excludeStories: /.*Data$/,
};

export const Default = () => <LoadingWidget />;
