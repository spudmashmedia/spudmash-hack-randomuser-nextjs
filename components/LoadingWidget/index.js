/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Spudmash Media Pty Ltd
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

import LoadingWidget from "./LoadingWidget";
export default LoadingWidget;
