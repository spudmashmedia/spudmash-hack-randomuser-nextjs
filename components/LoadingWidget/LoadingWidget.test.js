import React from "react";
import LoadingWidget from "./LoadingWidget";
import renderer from "react-test-renderer";

describe("LoadingWidget Component Snapshot Test", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<LoadingWidget />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
