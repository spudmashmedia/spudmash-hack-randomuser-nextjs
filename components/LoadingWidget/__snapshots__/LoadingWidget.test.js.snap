// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingWidget Component Snapshot Test renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
